src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
```

Comment:
None really elegant.
The only bullet proof solution would be to ignore the fact that it has to be the first statement and compare all lines if they contain the needle...
We could also `preg_replace_all` the whole file to check if any `declare(strict_types=1);` is declared and replace that one out then.
something like:
```php
// Match any line, which only contains of leading whitespace, "declare(strict_types=1);" suffixed by whitespace or comment.
$code = preg_replace(
    '/\r?\n[\t ]*(declare\(\s*strict_types\s*=\s*[^)]+\);)(?:[\t ]+\/[\/\*][^\r\n]+)?[\t ]*\r?\n/',
    '',
    $code
);
```
Which matches any line where there is only(!) `declare(strict_types=1);` and whitespace on it.
This matches:
```php
<?php
/*
 * [....]
 */

   declare(strict_types=1);    
```
But sadly also matches:
```php
<?php
/*
 * [....]
 */

$GLOBALS['TL_DCA']['tl_test'] = [
];

   declare(strict_types=1);    
```

However this would indicate that the initial file was already broken and trying therefore we can safely remove that one as well.

The last culprit for the matches is, that it also matches entities that are on a own line within a comment, but I think of this as a non issue:
```php
<?php
/*
   declare(strict_types=1);    
 */
```

On a performance aspect, I think we can marginalize that we are regexing the whole file because the execution will only be performed on container build time where micro optimization is not needed.